src/main/index.ts

```diff
@@ -122,9 +128,18 @@ app.on("ready", async () => {
 
   PrometheusProviderRegistry.createInstance();
   initializers.initPrometheusProviderRegistry();
+  syncGeneralEntities();
 
-  await initializers.initializeStores();
-  initializers.initializeWeblinks();
+  logger.info("💾 Loading stores");
+
+  UserStore.createInstance();
+  ClusterStore.createInstance().provideInitialFromMain();
+  HotbarStore.createInstance();
+  ExtensionsStore.createInstance();
+  FilesystemProvisionerStore.createInstance();
+  WeblinkStore.createInstance();
```

Comment:
Given that we are no longer awaiting anything the benefit of having a separate function is diminished (I think). But I can revert this to its own function if desirable.